wiki/en/misc/general.yml

```diff
@@ -1,5 +1,5 @@
 footer:
-  copyright:         Software by <a href="https://github.com/corrados" target="_blank" rel="noreferrer">Volker Fischer</a> and <a href="https://github.com/jamulussoftware/jamulus/graphs/contributors" target="_blank" rel="noreferrer">contributors</a>
+  copyright:         
```

Comment:
Remove this and all occurrences? But this implies we can’t translate the copyright

Reply:
I don't think we're allowed to translate it, are we? At least, the language given in the agreement is English. Interesting point though since the jurisdiction is Germany :-) But I assume if they had other legally binding translations, they'd have listed those.

Reply:
Oh sorry, this is ours, not ASIO's. Hm.